### PR TITLE
Fix spelling mistakes.

### PR DESCRIPTION
**Pull request by nick70**

### CODE REVIEW

File: src/entt/entity/registry.hpp [hunk 1]
```diff
@@ -775,7 +775,7 @@ class Registry {
      * }
      * @endcode
      *
-     * Prefer this function anyway because it has slighlty better
+     * Prefer this function anyway because it has slightlty better
```
Comment (skypjack) [inline src/entt/entity/registry.hpp]: It should be _slightly_, not _slightlty_. Can you fix it?

**Comment by nick70** [comment]
I don't know how I missed that lol

**Comment by reworks** [comment]
@nick70 You've accidentally left in the .vscode in the root of the repo.

**Comment by nick70** [comment]
Fixed.

**Comment by skypjack** [comment]
Thank you very much for your review. Really appreciated.
I't time to merge it upstream. Good job!!

Do not forget to star the project, so as to spread it more and more.

**Pull request merged**